plugins/README.md

```diff
@@ -36,6 +37,22 @@ Tips:
 -   To see a trace of logs and wasm ABI calls, add `--loglevel=TRACE`.
 -   Optionally specify plugin config data using the `--config=<path>` flag.
 
+# Building Plugins
+To build your plugin using bazel, run:
+
+```
```

Comment:
nit: use consistent formatting with other bazelisk command lines in this file, e.g.:

`$ bazelisk build <target>`

i.e. using single-ticks, and `$` in front of shell commands